data/assets.yaml

```diff
@@ -55,98 +55,104 @@ shapefiles:
     url: http://s3.amazonaws.com/tilezen-assets/curated/admin_areas_20180409.zip
 
   - name: ne_110m_lakes
-    url: http://www.naturalearthdata.com/http//www.naturalearthdata.com/download/110m/physical/ne_110m_lakes.zip
+    url: https://naciscdn.org/naturalearth/5.1.0/110m/physical/ne_110m_lakes.zip
```

Comment:
Thanks for switching these to versioned CDN links... much more obvious which version we're on and makes updating NE a deliberate process.

Reply:
I agree.  Much clearer

Reply:
These will all need to switch to 5.1.1 to pick up the other change.